backup_and_restore/application_backup_and_restore/oadp-use-cases/oadp-usecase-legacy-aws-plugin.adoc

```diff
@@ -0,0 +1,12 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-using-legacy-aws-plugin"]
+= Using the legacy-aws Velero plugin
+include::_attributes/common-attributes.adoc[]
+:context: oadp-using-legacy-aws-plugin
+
+toc::[]
+
+If you are using an {aws-short} S3-compatible backup storage location, you might get a `SignatureDoesNotMatch` error while backing up your application. This error occurs because, some backup storage locations still use the older versions of the S3 APIs, which are incompatible with the newer `AWS SDK for Go V2`. To resolve this issue, you can use the `legacy-aws` Velero plugin in the `DataProtectionApplication` custom resource (CR). The `legacy-aws` Velero plugin uses the older `AWS SDK for Go V1`, which is compatible with the legacy S3 APIs, ensuring successful backups.
```

Comment:
```suggestion
If you are using an {aws-short} S3-compatible backup storage location, you might get a `SignatureDoesNotMatch` error while backing up your application. This error occurs because some backup storage locations still use the older versions of the S3 APIs, which are incompatible with the newer `AWS SDK for Go V2`. To resolve this issue, you can use the `legacy-aws` Velero plugin in the `DataProtectionApplication` custom resource (CR). The `legacy-aws` Velero plugin uses the older `AWS SDK for Go V1`, which is compatible with the legacy S3 APIs, ensuring successful backups.
```
Do the phrases "AWS SDK for Go V2" and "AWS SDK for Go V1" need to be in back ticks? I don't think we typically highlight the names of SDKs in back ticks.

Reply:
Thank you Laura. I have removed the back ticks.